packages/server/graphql/types/StartSprintPokerPayload.ts

```diff
@@ -24,6 +24,11 @@ export const StartSprintPokerSuccess = new GraphQLObjectType<any, GQLContext>({
     },
     teamId: {
       type: new GraphQLNonNull(GraphQLID)
+    },
+    hasGcalError: {
```

Comment:
Did you consider sending snackbar notification instead? No strong preference, just thinking

Reply:
I am sending a snackbar notification if `hasGcalError` is true 😄